apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/SubscriberClient.cs

```diff
@@ -1099,6 +1101,10 @@ private async Task ProcessPullMessagesAsync(List<ReceivedMessage> msgs, HashSet<
                             _softStopCts.Token.ThrowIfCancellationRequested();
                             _userHandlerInFlight += 1;
                         }
+                        if (msg.DeliveryAttempt > 0)
```

Comment:
Okay, so this is part of a ReceivedMessage in the proto, and we're putting it into the PubsubMessage, which will be reused.

Reply:
ack